macros/src/lib.rs

```diff
@@ -287,3 +288,19 @@ fn is_result_unit(output: &ReturnType) -> bool {
         },
     }
 }
+
+#[proc_macro_derive(HotPatchMigrate)]
+pub fn derive_hot_patch_migrate(input: TokenStream) -> TokenStream {
+    let input = parse_macro_input!(input as DeriveInput);
+    let name = &input.ident;
+
+    let expanded = quote! {
+        impl bevy_simple_subsecond_system::migration::HotPatchMigrate for #name {
+            fn current_type_id() -> std::any::TypeId {
+                bevy_simple_subsecond_system::dioxus_devtools::subsecond::HotFn::current(|| std::any::TypeId::of::<Self>()).call(())
```

Comment:
Nit: Haven't familiarized myself with this crate yet, but at least for Bevy, we prefer pulling from `core` so we're more no_std-ready

```suggestion
            fn current_type_id() -> core::any::TypeId {
                bevy_simple_subsecond_system::dioxus_devtools::subsecond::HotFn::current(|| core::any::TypeId::of::<Self>()).call(())
```

Reply:
Moved to core and bevy platform.